applications/ShallowWaterApplication/custom_python/add_custom_utilities_to_python.cpp

```diff
@@ -63,18 +61,36 @@ namespace Python
         .def("CheckDryPrimitiveVariables", &ShallowWaterVariablesUtility::CheckDryPrimitiveVariables)
         .def("SetDryWetState", &ShallowWaterVariablesUtility::SetDryWetState)
         .def("DeactivateDryElements", &ShallowWaterVariablesUtility::DeactivateDryElements)
-        .def("DefineDryProperties", &ShallowWaterVariablesUtility::DefineDryProperties)
```

Comment:
I think that this belongs to another PR (just reporting it in case you didn't note it was added to this one).

Reply:
Belongs to that PR.
I don't need those methods anymore with the new implementation.

Reply:
the new utility manages several properties to separate in different layers the topography, the wet domain and the dry domain. The picture represents the run-up of a wave:

![Captura de pantalla de 2019-05-03 12-48-50](https://user-images.githubusercontent.com/25640870/57139287-2f34ac00-6db5-11e9-9d67-1cafed2e218a.png)

Reply:
Now it is clear to me. Sorry for the inconveniences.